environments/staging/inventory.ini.j2

```diff
@@ -126,6 +139,10 @@ airflow1
 # Amazon EC2
 {{ control-staging }} hostname=control-staging ec2=yes
 
+[control0]
+# Amazon EC2
+{{ control0-staging }} hostname=control0-staging ec2=yes
```

Comment:
I'm guessing you'll be seeing issues with python2 not being installed and with the private interface not being called "eth0". You can fix these problems by adding `ec2=ena ansible_python_interpreter=/usr/bin/python3`.

Until we find a better solution, all inventory entries for bionic hosts should include those vars (`ec2=ena ansible_python_interpreter=/usr/bin/python3`)